### PR TITLE
chore: update blog post

### PR DESCRIPTION
**Pull request by huanjani**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v123.en.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ image_height: '747'
 
 # AWS Copilot v1.23: App Runner Private Services, Aurora Serverless v2 and more!
 
-Posted On: Oct 31, 2022
+Posted On: Nov 1, 2022
```
Comment (Lou1415926) [inline site/content/blogs/release-v123.en.md:12]: Can we also update 
```
**Support Aurora Serverless v2 in `storage init`**: [See detailed section](#support-aurora-serverless-v2-in-storage-init).
```
to a link to the storage init page? 🥺

File: site/content/blogs/release-v123.en.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ http:
 By default, your private service can only send traffic to the internet.
 If you'd like to a send traffic to your environment, set [`network.vpc.placement: 'private'`](../../docs/manifest/rd-web-service/#network-vpc-placement) in your manifest.
 
-## Support Aurora Serverless v2 in `storage init`
+## Support Aurora Serverless v2 in [`storage init`](../docs/commands/storage-init.en.md)
```
Comment (dannyrandall) [inline site/content/blogs/release-v123.en.md:55]: i wonder if we should just remove this section down here?

Comment (huanjani) [reply site/content/blogs/release-v123.en.md:55]: Details for the "detailed section" coming this afternoon!

Comment (dannyrandall) [reply site/content/blogs/release-v123.en.md:55]: chatted offline - adding details in a separate PR :)

**Review by dannyrandall — APPROVED** [review]

**Review by Lou1415926 — APPROVED** [review]

**Pull request merged**